bson/src/test/unit/org/bson/json/JsonReaderTest.java

```diff
@@ -119,6 +132,16 @@ public void testDateTimeShell() {
     }
 
     @Test
+    public void testDateTimeShellNow() {
+        long currentTime = new Date().getTime();
+        String json = "ISODate()";
```

Comment:
MongoDB shell version: 3.0.7
connecting to: test
Server has startup warnings:
2015-10-27T16:24:54.175-0400 I CONTROL  [initandlisten]
2015-10-27T16:24:54.175-0400 I CONTROL  [initandlisten] ** WARNING: soft rlimits too low. Number of files is 256, should be at least 1000
> d = new Date()
ISODate("2015-10-29T13:25:02.432Z")
> d = new ISODate()
ISODate("2015-10-29T13:25:54.381Z")

They look identical.